org.goldendict.desktop.yml

```diff
@@ -0,0 +1,146 @@
+
+  - name: eb
+    config-opts:
+      - '--enable-shared'
+      - '--disable-static'
+    cleanup:
+      - /bin
+      - /share/doc
+    sources:
+      - type: git
+        url: https://salsa.debian.org/debian/eb.git
+        tag: debian/4.4.3-13
+        x-checker-data:
+          type: anitya
+          project-id: 155542
+          tag-template: debian/$version
+      # Update config.guess & config.sub to allow builds for aarch64 and others
+      # https://salsa.debian.org/debian/eb/-/merge_requests/3
+      - type: file
+        url: https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=6faca61810d335c7837f320733fe8e15a1431fc2
+        dest-filename: config.guess
+        sha256: 7d1e3c79b86de601c3a0457855ab854dffd15163f53c91edac54a7be2e9c931b
+      - type: file
+        url: https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=6faca61810d335c7837f320733fe8e15a1431fc2
+        dest-filename: config.sub
+        sha256: b1016a1eb11cdf0903bbc533e42e8ac2cc4d54a80b0545f4f0af5a85198277e2
+
+  - name: zstd
+    buildsystem: meson
+    subdir: build/meson
+    config-opts:
+      - -Dbin_programs=false
+      - -Dbin_contrib=false
+    sources:
+      - type: git
+        url: https://github.com/facebook/zstd.git
+        tag: v1.4.8
+        x-checker-data:
+          type: anitya
+          project-id: 12083
+          tag-template: v$version
+
+  - name: libao
+    cleanup:
+      - /include
+      - '*.la'
+      - /lib/pkgconfig
+      - /lib/ckport
+      - /share
+    sources:
+      - type: archive
+        url: 'https://downloads.xiph.org/releases/ao/libao-1.2.0.tar.gz'
+        sha256: 03ad231ad1f9d64b52474392d63c31197b0bc7bd416e58b1c10a329a5ed89caf
+        x-checker-data:
+          type: anitya
+          project-id: 7629
+          url-template: 'https://downloads.xiph.org/releases/ao/libao-$version.tar.gz'
+
+  - name: goldendict
+    buildsystem: qmake
+    build-options:
+      env:
+        CC: 'ccache gcc'
+        CXX: 'ccache g++'
+        QMAKEPATH: '/app/lib'
+        LD_LIBRARY_PATH: ${LD_LIBRARY_PATH}:/app/lib/:/usr/lib
+    config-opts:
+      - DEFINES+=USE_DBUS
+      - INCLUDEPATH+="${FLATPAK_DEST}/include/QtWebKitWidgets"
+      - INCLUDEPATH+="${FLATPAK_DEST}/include/QtWebKit"
+      - CONFIG+=release
+    build-commands:
+      # https://github.com/goldendict/goldendict/pull/1354
+      - desktop-file-edit --set-icon=${FLATPAK_ID} redist/goldendict.desktop
+      # Put the version to the metadata
+      - export VERSION=$(git rev-parse --short HEAD) # The same version as the upstream
+      - appstream-util modify redist/goldendict.appdata.xml releases "<release version=\"`cat version.txt` (${VERSION}) \" date=\"`date --iso-8601`\"/>"
+      - sed -i -e "s,<releases>&lt;,<releases><," -e "s,&gt;</releases>,></releases>," redist/goldendict.appdata.xml
+
+      - make -j$FLATPAK_BUILDER_N_JOBS
+      - make -j$FLATPAK_BUILDER_N_JOBS install
```

Comment:
Doesn't it mean you run make and make install twice here? `qmake` buildsystem should already do that

Reply:
Wow, great to know.